l10n_co_geo/data/res_country_state.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
OK, I see that this country is not yet considered to be packed as ZIP. This is because low demand. I don't know the mechanisms for uploading or demanding this pack. Maybe you can ask in geonames itself. Then continue this way until there's a better solution.